examples/howto/howto_debugging.ipynb

```diff
@@ -4,332 +4,537 @@
    "cell_type": "markdown",
```

Comment:
<p>These are supposed to reflect when it was originally written and by whom. If you find the date in the git history it should go here, same with the author (although in this case I'm pretty sure it was me). Usually upgrading/porting a NB does not add to authors but you added new information here so you should be co-author.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/406/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
@twiecki - thanks! I asked who authored the original in the issue tracker but didn't get a reply, I didn't think about commit history. the last commit I found was by MarcoGorelli committed on Dec 14, 2020, but it was a move from pymc3. I can't figure out how to track the original. I can put your name and Dec 14, 2020 as the date, or let me know if I should use an earlier date. I don't think I added much, just re-dressed the ideas that you put in (but figured that folks should know whom to blame since I couldn't find the original author :-) ).

Reply:
@twiecki - new changes are in, thanks to @reshamas  who found the commit history.